try_task_config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "parameters": {
+        "optimize_target_tasks": true,
+        "target_tasks_method": "nightly",
+        "version": "2021.11.23-nightly-stg"
+    },
+    "version": 2
+}
```

Comment:
Whats this used for?

Reply:
Looks like you were using it to trigger nightly tasks? Which means you wouldn't want to check that in :) Maybe Aki or Ahal told you to do it this way, but what I've done for Focus is to push my pr to the staging repo (in your case staging-fenix) and trigger the nightly cron task via the Taskcluster UI so you're not actually triggering a production push-apk task for nightly.

Reply:
We may want to adjust https://github.com/mozilla-mobile/shared-docs/blob/main/android/taskcluster_guide.md#how-do-i-run-a-staging-nightlybetarelease-on-prs to document the other way of triggering staging releases (which is via staging repos). Or maybe we document that in taskgraph docs and point the url to that instead?

Reply:
oh, didn't even know about that way of testing :) so this file just needs to be removed then. But yeah, mentioning the other way in the docs is a good idea!

Reply:
Taskgraph is a pre-requisite to allow this type of testing so with your recent work there, we're now able to trigger staging releases easier there too, another nice side-effect from that 👍 

Which reminds me we could improve the docs to reflect this:
* for new repos without taskgraph - forking is probably smoother and easier
* for existing taskgraph repos - triggering staging releases in PRs is faster I believe?

Reply:
> oh, didn't even know about that way of testing :) so this file just needs to be removed then. But yeah, mentioning the other way in the docs is a good idea!

+1, we shouldn't land this.

Reply:
It's for running a staging release (I'll have to remove this file before merging)
More info [here](https://github.com/mozilla-mobile/shared-docs/blob/main/android/taskcluster_guide.md#how-do-i-run-a-staging-nightlybetarelease-on-prs)

Reply:
^ Ignore my reply. GitHub only pushed to browser the first comment from sarah and not the rest of the conversion.